.github/workflows/mobilecoin-dispatch-dev-delete.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+#
+# MobileCoin Core projects - Dispatch (manual) Job - Delete target dev namespace and all of its components.
+
+name: mobilecoin-dispatch-dev-delete
+
+on:
+  workflow_dispatch:
+    inputs:
+      namespace:
+        description: "Target Namespace"
+        type: string
+        required: true
+
+jobs:
+  delete:
+    uses: ./.github/workflows/mobilecoin-workflow-dev-delete.yaml
+    with:
+      namespace: ${{ github.event.inputs.namespace }}
+    secrets:
```

Comment:
Do we not need to pass in the two `LEDGER_AWS_*` secrets to the mobilecoin-workflow-dev-delete.yaml?  They are marked as required.  I thought you could not access the repo secrets from a workflow_call unless they were passed in.

Reply:
Yeah good catch.  The delete workflows have not been thoroughly tested yet.